src/middlewared/middlewared/plugins/system.py

```diff
@@ -152,6 +155,323 @@ def debug(self, job):
+        ip4_addresses_list = [alias_dict['address'] for alias_dict in ip_addresses if alias_dict['type'] == 'INET']
+        ip6_addresses_list = [alias_dict['address'] for alias_dict in ip_addresses if alias_dict['type'] == 'INET6']
+
+        ip4_address = data.get('ui_address')
+        if (
+            ip4_address and
+            ip4_address != '0.0.0.0' and
+            ip4_address not in ip4_addresses_list
+        ):
+            verrors.add(
+                f'{schema}.guiaddress',
+                'Selected ipv4 address is not associated with this machine'
+            )
+
+        ip6_address = data.get('ui_v6address')
+        if (
+            ip6_address and
+            ip6_address != '::' and
+            ip6_address not in ip6_addresses_list
+        ):
+            verrors.add(
+                f'{schema}.guiv6address',
+                'Selected ipv6 address is not associated with this machine'
+            )
+
+        syslog_level = data.get('sysloglevel')
+        if not syslog_level:
+            verrors.add(
+                f'{schema}.sysloglevel',
+                'This field is required'
+            )
+
+        syslog_server = data.get('syslogserver')
+        if not syslog_server:
+            verrors.add(
+                f'{schema}.syslogserver',
+                'This field is required'
+            )
+        else:
+            match = re.match("^[\w\.\-]+(\:\d+)?$", syslog_server)
+            if not match:
+                verrors.add(
+                    f'{schema}.syslogserver',
+                    'Invalid syslog server format'
+                )
+            elif ':' in syslog_server:
+                port = int(syslog_server.split(':')[-1])
+                if port < 0 or port > 65535:
+                    verrors.add(
+                        f'{schema}.syslogserver',
+                        'Port specified should be between 0 - 65535'
+                    )
+
+        protocol = data.get('ui_protocol')
+        if not protocol:
+            verrors.add(
+                f'{schema}.guiprotocol',
+                'This field is required'
+            )
+        else:
+            if protocol != 'http':
+                certificate_id = data.get('ui_certificate')
+                if not certificate_id:
+                    verrors.add(
+                        f'{schema}.guicertificate',
+                        'Protocol has been selected as HTTPS, certificate is required'
+                    )
+                else:
+                    # getting fingerprint for certificate
+                    fingerprint = await self.middleware.call(
+                        'certificate.get_fingerprint',
+                        certificate_id
+                    )
+                    if fingerprint:
+                        syslog.openlog(logoption=syslog.LOG_PID, facility=syslog.LOG_USER)
+                        syslog.syslog(syslog.LOG_NOTICE, 'Fingerprint of the certificate used in UI : ' + fingerprint)
```

Comment:
@sonicaj lets use LOG_ERR here so it ends up in console and /var/log/messages

Reply:
*.err;kern.warning;auth.notice;mail.crit		/dev/console
*.notice;authpriv.none;kern.debug;lpr.info;mail.crit;news.err	/var/log/messages
@william-gr all error messages go to the console ?

Reply:
@sonicaj all ERR messages go to console, not NOTICE